packages/App/src/store/reducers/Wallet.tsx

```diff
@@ -106,9 +133,15 @@ const Wallet = createReducer<IWallet>(initialState, builder => {
         return state;
       },
     )
+    .addCase(filterAllItemsRequest, state => ({
+      ...state,
+      isLoadingTransactions: true,
+      errorFilters: '',
```

Comment:
You should write your reducers in one fashion: either use immutable style or mutate store. This should be consistent